### PR TITLE
deprecating server-side direction computation

### PR DESCRIPTION
**Pull request by melange396**

### CODE REVIEW

File: src/acquisition/covidcast/database.py [hunk 1]
```diff
@@ -106,6 +106,7 @@ def insert_or_update_batch(self, cc_rows, batch_size=2**20, commit_partial=False
 
     tmp_table_name = 'tmp_insert_update_table'
 
+    # TODO: this heavily copypastas src/ddl/covidcast.sql -- theres got to be a better way
```
Comment (benjaminysmith) [inline src/acquisition/covidcast/database.py:109]: Is it worth creating an issue for tracking this or do  you typically leave as todo?

**Review by benjaminysmith — APPROVED** [review]
I love seeing deleted code!

It looks like this changes the API -- I'm guessing we leave the data in the DB?

**Comment by krivard** [comment]
@benjaminysmith Yep, we'll leave the data in the DB until we're ready to do a database migration. We're not as good at doing that without substantial service interruptions, so it will probably need to wait until we're less in the PR spotlight from the mask data release from Monday.

**Review by krivard — APPROVED** [review]
Looks great!

File: src/acquisition/covidcast/database.py (hunk 1 above)
Comment (krivard) [reply src/acquisition/covidcast/database.py:109]: Leaving a TODO essentially creates a lowest-priority task. If it becomes more obviously important we can escalate to an issue.

**Pull request merged**